pkg/cli/config/config.go

```diff
@@ -17,11 +17,12 @@ import (
 
 // Please update the validateConfig function if there is any required config key added
 type BackplaneConfiguration struct {
-	URL              string  `json:"url"`
-	ProxyURL         *string `json:"proxy-url"`
-	SessionDirectory string  `json:"session-dir"`
-	AssumeInitialArn string  `json:"assume-initial-arn"`
-	PagerDutyAPIKey  string  `json:"pd-key"`
+	URL              string   `json:"url"`
+	ProxyURL         *string  `json:"proxy-url"`
+	ProxyURLs        []string `json:"proxy-urls"`
```

Comment:
Why do we need two configs?

Reply:
@samanthajayasinghe  - My intention is to load all the Proxy urls from the local backplane configuration just as below:
```
Checking proxy connectivity...
Loading proxy URLs from local backplane configuration:
[http://squid.corp.redhat.com:3128; http://proxy2.squi-001.prod.rdu2.dc.redhat.com:3128]
```
And the current `ProxyURL *string "json:\"proxy-url,omitempty\"" ` field cannot handle multiple proxy URLs directly. 

Given that removing the `ProxyURL` field can causes issues with existing functions, I think it might be better to keep it temporarily to ensure backward compatibility, we can refactor later on.

Reply:
I guess it's good to keep `proxy-URL` to accept multiple URLs, which keeps the backward compatibility.
If it really need, we will refactor it via another card/PR  
What do you think?